src/test/java/fr/socrates/domain/attendee/ConfirmationServiceImplTest.java

```diff
@@ -91,4 +93,16 @@ public void should_save_confirmation_payment() throws Exception {
         Confirmation confirmationExpected = new Confirmation(new CandidateId(email), now, AccommodationChoice.DOUBLE_ROOM, Payment.AT_CHECKOUT);
         assertThat(confirmationService.getListConfirmations()).containsExactly(confirmationExpected);
     }
+
+    @Test
+    public void should_add_a_vegan_diet_for_a_given_participant() throws Exception {
```

Comment:
It's not an unit test cause you depend on adding a candidate and confirming the candidate. It's an acceptance test...